sdlib/d/gc/hpd.d

```diff
@@ -111,7 +111,7 @@ public:
 		return bestIndex;
 	}
 
-	void release(uint index, uint pages) {
+	void shrinkAlloc(uint index, uint pages) {
```

Comment:
This doesn't deal with an allocation or shrinking anymore, as it turns out. It simple clears some pages. That would be a better name. not a blocker, but worth thinking about as we go through semantic compression.